### DISCUSSION TITLE
Welcome to PureCSS Discussions!

### DISCUSSION THREAD

**Discussion by redonkulus** [pure-css/pure#831] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cooperdozier** [comment]
Hi! I use a subset of PureCSS on my site savetheoxygen.org . I used to use it on a personal website, but the site never had a clear purpose so when the registration lapsed I let it go. Also used it on a little freelance project in 2013.

**Comment by menel972** [comment]
Hi 👋🏾
I'm testing pure-menu and I dont know how to change the background's color when you hover over the element with the cursor.

I tried "li/a/ul:hover{}" but it doesn't work. 

Can you help me ?

**Reply by redonkulus** [reply]
You need to put it on the "anchor" element ... `ul li a:hover { color: red }`

**Reply by menel972** [reply]
Thank you for the answer, I did that for the font color and it worked, but this is the background color I want to change.

li:hover{background-color : red} (for exemple) doesnt work

**Reply by redonkulus** [reply]
you tried `li:hover { background-color: blue; }`?

**Reply by menel972** [reply]
Yes, it doesnt work

**Reply by redonkulus** [reply]
that should work, you may have some conflicting style that is causing issues.

**Reply by menel972** [reply]
![image](https://user-images.githubusercontent.com/83010883/116927770-636d2c80-ac5c-11eb-98b2-61cae43f4781.png)
![image](https://user-images.githubusercontent.com/83010883/116927779-6700b380-ac5c-11eb-8e4b-d9257c5a867d.png)

The background still grey 😕

**Reply by redonkulus** [reply]
You can try putting the background color on the anchor ... `li a:hover { background-color: blue }`

**Reply by menel972** [reply]
I'll try this tomorrow 

Thanks for your help ! 🙏🏾

**Reply by menel972** [reply]
Hi !

I’ve finally succeeded. 
I had to use the selector ".pure-menu-link" instead of "li".

![image](https://user-images.githubusercontent.com/83010883/116982497-268b4f00-acc9-11eb-9be7-3fc809ac8ba3.png)

**Reply by cooperdozier** [reply]
Yeah, I have 
```
a.pure-menu-link:hover {
    border-radius: 4px;
    border: solid 0.1em #426b96;
}
```
and I find in the Pure rules I use (savetheoxygen.org) 
```
.pure-menu-active>.pure-menu-link,
.pure-menu-link:focus,
.pure-menu-link:hover {
	background-color: #eee
}
```
since class selectors have higher specificity than element selectors, they will override the element selector, even if the element selector is lower down in the CSS file.

**Reply by menel972** [reply]
Good to know ! Thx
